node-backend/inProgressRepositoryService.ts

```diff
@@ -60,15 +60,17 @@ export function mapWorkspace(
 export function mapCriteria(criteria: ICriterionQueryResult[]): ICriterion[] {
   return _(criteria)
     .sortBy('orderindex')
-    .map((queryCriterion) => {
-      return {
-        id: queryCriterion.id,
-        title: queryCriterion.title,
-        description: queryCriterion.description,
-        isFavourable: queryCriterion.isfavourable,
-        dataSources: []
-      };
-    })
+    .map(
+      (queryCriterion): ICriterion => {
```

Comment:
`queryCriterion` isn't typed; might be unnecesary due to lodash generics? But looks odd to me

Reply:
Yes, but typed it anyway now :)